Disable symbolize_test, which fails to link on amd64 where ld.lld
defaults to --exec-only.

Index: absl/debugging/CMakeLists.txt
--- absl/debugging/CMakeLists.txt.orig
+++ absl/debugging/CMakeLists.txt
@@ -92,6 +92,7 @@ absl_cc_library(
   PUBLIC
 )
 
+if (NOT CMAKE_SYSTEM_NAME STREQUAL "OpenBSD")
 absl_cc_test(
   NAME
     symbolize_test
@@ -114,6 +115,7 @@ absl_cc_test(
     absl::symbolize
     GTest::gmock
 )
+endif()
 
 # Internal-only target, do not depend on directly.
 absl_cc_library(
